File: js/delivery-api/GetItems.js
// DocSection: delivery_api_get_items
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
const KontentDelivery = require('@kontent-ai/delivery-sdk');

const deliveryClient = KontentDelivery.createDeliveryClient({
  projectId: '<YOUR_PROJECT_ID>'
});

const response = await deliveryClient.items()
  .limitParameter(3)
  .toPromise();
// EndDocSection

File: ruby/delivery-api/GetItems.rb
# DocSection: delivery_api_get_items
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '<YOUR_PROJECT_ID>'
delivery_client.items('system.type'.eq('article'))
               .limit(3)
               .execute do |response|
                 items = response.items
               end
# EndDocSection